src/vs/workbench/services/extensionManagement/browser/extensionBisect.ts

```diff
@@ -20,7 +20,7 @@ import { Extensions, IWorkbenchContributionsRegistry } from 'vs/workbench/common
 import { ICommandService } from 'vs/platform/commands/common/commands';
 import { ILogService } from 'vs/platform/log/common/log';
 import { IProductService } from 'vs/platform/product/common/productService';
-import { IWorkbenchIssueService } from 'vs/workbench/services/issue/common/issue';
+import { IWorkbenchIssueService } from 'vs/workbench/contrib/issue/common/issue';
```

Comment:
We need to fix this layer violation. I suggest to `ICommandService.executeCommand` with https://github.com/justschen/vscode/blob/a19cc3b3f1c0c22227fee44d80e16d44e103247e/src/vs/workbench/contrib/issue/common/issue.contribution.ts#L74 to break this issue.